src/number/format-decimal.js

```diff
@@ -19,6 +19,15 @@ module.exports = function(value, localeData, options) {
 	var ret = '';
 	var processedDecimal = (valueStr.indexOf('.') < 0);
 	var digitNum = 0;
+	var currentGroupSize ;
+	var groupSizes;
+	if ( Array.isArray( localeData.groupSize) ) {
+		groupSizes = localeData.groupSize.slice(); 	// make a copy of the groupsizes
+		currentGroupSize = groupSizes.shift();		// pop the first groupSize from the front
+	} else {
+		groupSizes = [];
+		currentGroupSize = localeData.groupSize;
+	}
```

Comment:
Sure no problem. I used that solution originally, but I thought this was more succinct.  I am by no means a Javascript expert however, and I defer to your guys' judgment.